citest/azure_testing/az_agent.py

```diff
@@ -0,0 +1,41 @@
+# Base agent for Azure CITest scenario
+# pylint: disable=W0311
+
+"""Adapts citest CliAgent to interact with Microsoft Azure using AZ 2.0 CLI"""
```

Comment:
Can you add a link to documentation instructing how to obtain, install, and configure the cli (if external configuration is required)?